.github/workflows/charm-update-libs.yaml

```
@@ -27,26 +27,87 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Fetch charm libraries
+      - name: Merge update-libs PRs with green CI
```

Not everyone will kow what 'update-libs' PRs are, maybe we can be more explicit here.
How about:
```suggestion
      - name: Merge any PRs from the automatically generated "chore/auto-libs" branch if the CI is green
```